core/api/v1/keytransparency.proto

```diff
@@ -65,7 +66,13 @@ message Entry {
   // modifying creating a hash chain of all mutations. The hash used is
   // CommonJSON in "github.com/benlaurie/objecthash/go/objecthash".
   bytes previous = 8;
+}
 
+// SignedEntry is a cryptographically signed Entry.
+// SignedEntry will be storead as a trillian.Map leaf.
+message SignedEntry {
```

Comment:
Do we want to introduce an enum indicating what signature scheme is used? Or is that specified in the directory?

Reply:
The signature scheme is specified by the public key itself. (This is one of the nice things about TINK public keys)